src/services/near-nep21-util.js

```diff
@@ -204,8 +227,8 @@ export function normalizeAmount(value) {
 // 0 => "0".repeat(24)
 // The returned string will have at least 24 zeros
 //alias to normalizeAmount
-export const toYoctosString = normalizeAmount;
-// function toYoctosString(amount){
+export const toNumber = normalizeAmount;
+// function toNumber(amount){
```

Comment:
do we need this? Can we remove `toNumber` and use `normalizeAmount` instead?

Reply:
yes, removing